From 35a5bc12d48384c01b47fca62de7652209042393 Mon Sep 17 00:00:00 2001
From: Jordan Williams <jordan@jwillikers.com>
Date: Sun, 4 Jun 2023 10:58:09 -0500
Subject: [PATCH] Use a tilde to separate pre-release versions in the AppData

This is required for properly ordering the version numbers.
Without this, the version ordering is deemed invalid.
The tilde character is recommended for this case: https://www.freedesktop.org/software/appstream/docs/chap-AppStream-Misc.html#spec-vercmp-recommendations
This causes the Flatpak to fail to build currently: https://buildbot.flathub.org/#/builders/31/builds/3925

AppData can be validated locally according to the instructions here: https://github.com/flathub/flathub/wiki/AppData-Guidelines#use-flathubs-appstream-util
---
 packaging/linux/org.thonny.Thonny.appdata.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/packaging/linux/org.thonny.Thonny.appdata.xml b/packaging/linux/org.thonny.Thonny.appdata.xml
index 5ee0545c..4926c53b 100644
--- a/packaging/linux/org.thonny.Thonny.appdata.xml
+++ b/packaging/linux/org.thonny.Thonny.appdata.xml
@@ -51,7 +51,7 @@
         </p>
       </description>
     </release>
-    <release version="4.1.0b1" date="2023-04-23">
+    <release version="4.1.0~b1" date="2023-04-23">
       <description>
         <p>
           Feature pre-release.
